File: Richest Customer Wealth/Richest Customer Wealth.py
class RichestCustomerWealth:

    def get_maximumWealth(self,accounts):
        result_lst = 0
        for i in accounts:
            sum =0
            if i is list:
                for j in i:
                    sum+=j
                if sum>result_lst:
                    result_lst = sum
        return result_lst

ans = RichestCustomerWealth()
accounts = [[1, 2, 3], [3, 2, 1]]
print(ans.get_maximumWealth(accounts))

File: Richest Customer Wealth/RichestCustomerWealth.java
import java.util.Arrays;

public class RichestCustomerWealth {
    public int[] getmaximumwealth(int [][] accounts)
    {

        int maxwealt = 0;
        for(int [] element: accounts)
        {
            int currentCustomerWealth = 0;
            for(int bank: element)
            {
                currentCustomerWealth+=bank;
            }
            maxwealt = Math.max(maxwealt,currentCustomerWealth);

        }
        return new int[]{maxwealt};
    }

    public static void main(String[] args)
    {
        int [] []arr = {{1,2,3},{1,5,1}};
        RichestCustomerWealth richestCustomerWealth = new RichestCustomerWealth();
        System.out.println(Arrays.toString(richestCustomerWealth.getmaximumwealth(arr)));

    }

}
